node/rust-client/src/lib.rs

```diff
@@ -58,6 +64,27 @@ struct AsyncClient {
     runtime: Runtime,
 }
 
+#[napi(object)]
+#[derive(Clone)]
+pub struct OpenTelemetryTracesConfig {
+    pub endpoint: Option<String>,
```

Comment:
same - endpoint shouldn't be optional , the whole struct should be optional in OTEL config

Reply:
also - use the same names, above you called it "exporter"